prdoc/prdoc/pr_6715.prdoc

```diff
@@ -0,0 +1,14 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: Update Nomination Pool Pallet to Support Block Number Provider
+
+doc:
+  - audience: Runtime Dev
+    description: |
+      This PR makes the nomination pool pallet use the relay chain as a block provider for a parachain on a regular schedule.
```

Comment:
It doesn't force the nomination pool pallet to use the relay chain block provider,
```suggestion
      This PR makes the block number provider used in the nomination pool pallet configurable. Before this PR, nomination pool pallet always used the system block number, with this PR some runtime can opt to use the relay chain block number instead.
```